source/getting-started/quickstart.md

```diff
@@ -0,0 +1,262 @@
+
+## Create a New Application
+
+Once you've installed Ember via npm, you will have access to a new
+`ember` command in your terminal. You can use the `ember new` command to
+create a new application.
+
+```sh
+ember new ember-quickstart
+```
+
+This one command will create a new directory called `ember-quickstart`
+and set up a new Ember application inside of it.  Out of the box, your
+application will include:
+
+* A development server
+* Template compilation
+* CSS preprocessing
+* JavaScript and CSS minification
+* ES2015 features via Babel
+
+By providing everything you need to build production-ready web
+applications in an integrated package, Ember makes starting new projects
+a breeze.
+
+Let's make sure everything is working properly. Open the
+`ember-quickstart` directory in your favorite editor, such as Sublime
+Text or Vim. Once you have your editor open, `cd` into the application
+directory and start the development server by typing:
+
+```sh
+cd ember-quickstart
+ember serve
+```
+
+After a few seconds, you should see output that looks like this:
+
+```text
+Livereload server on http://localhost:49152
+Serving on http://localhost:4200/
+```
+
+(To stop the server at any time, type Ctrl-C in your terminal.)
+
+Open [http://localhost:4200/](http://localhost:4200) in your browser of
+choice. You should see a page that says "Welcome to Ember" and not much
+else. Congratulations! You just created and booted your first Ember app.
+
+Switch to your editor and open `app/templates/application.hbs`. This is
+called the `application` template and it is always on screen while the
+user has your application loaded.
+
+In your editor, change the text inside the &lt;h1&gt; from `Welcome to
+Ember` to `PeopleTracker` and save the file. Notice that Ember detects
+the change you just made and automatically reloads the page for you in
+the background. You should see that "Welcome to Ember" has been
+replaced by "PeopleTracker".
+
+## Define a Route
+
+Ember makes URLs the heart of your application, so you never build apps
+that feel broken to your users. Before doing anything in Ember, first think
+about your application's URLs.
+
+Let's build an application that shows a list of scientists. To do that,
+the first step is to create a route. For now, you can think of routes as
+being the different pages that make up your application.
+
+Ember comes with _generators_ that automate the boilerplate code for
+common tasks. To generate a route, type this in your terminal:
+
+```sh
+ember generate route scientists
+```
+
+You'll see output like this:
+
+```text
+installing route
+  create app/routes/scientists.js
+  create app/templates/scientists.hbs
+updating router
+  add route scientists
+installing route-test
+  create tests/unit/routes/scientists-test.js
+```
+
+That's Ember telling you that it has created:
+
+1. A template to be displayed when the user visits `/scientists`
+2. A `Route` object that fetches the model used by that template
```

Comment:
@kimroen I specifically wanted to use the word "model" to strengthen the mental model that "`model()` hook means template model".

@trek It's key for the refactoring step from route template to component template. I agree that this should be replaced by a routable component once those land.